pac4j-http/src/main/java/org/pac4j/http/credentials/authenticator/DigestAuthenticator.java

```diff
@@ -0,0 +1,15 @@
+package org.pac4j.http.credentials.authenticator;
+
+import org.pac4j.http.credentials.DigestCredentials;
+import org.pac4j.http.credentials.UsernamePasswordCredentials;
+
+/**
+ * This interface represents the contract to validate digest credentials.
+ *
+ * @author Mircea Carasel
+ */
+public interface DigestAuthenticator extends Authenticator<DigestCredentials> {
```

Comment:
That is correct, they do not play any specific role for username/password validation. I think I can get read of DigestAuthenticator interface completely.